srcpkgs/glslViewer/template

```diff
@@ -1,31 +1,28 @@
 # Template file for 'glslViewer'
 pkgname=glslViewer
-version=1.6.8
+version=3.0.7
 revision=1
-build_style=gnu-makefile
-make_use_env=yes
+_vera_gitrev="91c48874900909ebd58440359657948aed906f50"
+build_style=cmake
 hostmakedepends="pkg-config"
 makedepends="glfw-devel glu-devel MesaLib-devel libX11-devel libXrandr-devel
  libXi-devel libXxf86vm-devel libXcursor-devel libXinerama-devel libXext-devel
- libXrender-devel libXdamage-devel"
+ libXrender-devel libXdamage-devel ncurses-devel libatomic-devel liblo-devel"
```

Comment:
Is libatomic required always or only on armv6? If only on armv6, it should be added conditionally.

Reply:
libatomic is required always regardless of architecture.

I will note this moment in the commit.

Reply:
Okay, false alarm. libatomic has to be placed under a condition.

`target_link_libraries(glslViewer PRIVATE atomic)` should not require libatomic on x86_64-musl since it already has one.

Reply:
The problem is solved by patching CMakeLists.txt.

libatomic should not be required everywhere.

Reply:
Solved by removing `target_link_libraries(glslViewer PRIVATE atomic)` from CMakeLists when `$XBPS_TARGET_NO_ATOMIC8` is unset.